cg/models/cg_config.py

```diff
@@ -458,12 +462,29 @@ def pdc_api(self) -> PdcAPI:
             self.pdc_api_ = api
         return api
 
+    @property
+    def slurm_service(self) -> SlurmService:
```

Comment:
```suggestion
    def slurm_cli_service(self) -> SlurmCLIService:
```

Reply:
I'll leave this, the cli service implements the slurm service interface. This is intentional